### ISSUE TITLE
Hide reply box in forum post view when it is disabled by warning

### ISSUE THREAD

**Issue by kj415j45** [ppy/osu-web#6808]

**Comment by peppy** [comment]
Can you give an example screenshot of where you expect it to be hidden?

**Comment by kj415j45** [comment]
Here is the screenshot, where the reply box is in the red square.
![image](https://user-images.githubusercontent.com/18349191/97249826-98c53780-183f-11eb-94d3-6025c6634c0a.png)

**Comment by peppy** [comment]
Yeah, probably should change in one of the two mentioned ways. Right now it doesn't even look disabled at a glance.

**Comment by nanaya** [comment]
It is there in case someone have saved draft post in that thread.

**Comment by peppy** [comment]
Can we only show it if there's a draft, in that case?
